### DISCUSSION TITLE
How to hide header?

### DISCUSSION THREAD

**Discussion by roozbehzarei** [upptime/upptime#927] [Q&A] [answered]
I want to show the status page in my app, therefore I want to hide the header section (white background with logo and links) to avoid having multiple headers (the app already has a nav bar).
What file should I modify to achieve this goal?

**Comment by AnandChowdhary** [comment, marked as answer]
I think the easiest would be to add custom CSS and do `display: none` on the header: https://upptime.js.org/docs/configuration/#custom-css
